class/Defaults/Carrier/Webhook.php

```diff
@@ -217,7 +178,7 @@ public function send_post( $url, $args = [], $headers = [] ) {
 
 		$remote_args = apply_filters( 'notification/carrier/webhook/remote_args/post', $remote_args, $url, $args, $this );
```

Comment:
```suggestion
		$remote_args = apply_filters( 'notification/carrier/webhook/remote_args', $remote_args, $url, $method, $args, $this );
```

Make this filter more elastic. Make sure to put the old filter as deprecated.